staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -48,3 +48,47 @@ type AdmissionPluginConfiguration struct {
 	// +optional
 	Configuration *runtime.Unknown `json:"configuration"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ConnectivityServiceConfiguration provides versioned configuration for network proxy clients.
+type ConnectivityServiceConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// ConnectionServices contains a list of network proxy client configurations
```

Comment:
These names are lowercased to follow the json tags.  It generates openapi correctly.

Reply:
Confused by your comment. Not getting what you are trying to tell me (or ask me to change).

Reply:
Got, Fixed.